Tests/Reqnroll.RuntimeTests/StepExecutionTestsBase.cs

```diff
@@ -133,17 +133,17 @@ protected TestRunner GetTestRunnerFor(Action<IObjectContainer> registerMocks, pa
         {
             return TestObjectFactories.CreateTestRunner(
                 container =>
-                    {
-                        container.RegisterTypeAs<DummyTestTracer, ITestTracer>();
-                        container.RegisterInstanceAs(ContextManagerStub);
+                {
```

Comment:
No meaningful change here - just formatting that VS apparently insisted upon 😄

Reply:
Actually I guess while I'm here... I have a personal pet peeve on neglecting curly braces for `foreach`, `if`, etc, so I'm going to "fix" that below 🙂